File: data/museum/oseg_generated/typescript-node/DeleteSpecialEvent_default_example.ts
import * as fs from 'fs';
import * as openapimuseum_client from "openapimuseum_client";

const apiCaller = new openapimuseum_client.EventsApi();

const eventId = "dad4bce8-f5cb-4078-a211-995864315e39";

apiCaller.deleteSpecialEvent(
    eventId,
).catch(error => {
  console.log("Exception when calling Events#deleteSpecialEvent:");
  console.log(error.body);
});

File: examples/museum/oseg_generated/ruby/DeleteSpecialEventDefaultExample.rb
require "openapimuseum_client"

OpenApiMuseumClient.configure do |config|
end

begin
    OpenApiMuseumClient::EventsApi.new.delete_special_event(
        "dad4bce8-f5cb-4078-a211-995864315e39",
    )
rescue OpenApiMuseumClient::ApiError => e
    puts "Exception when calling Events#delete_special_event: #{e}"
end
